R/get_fs_int.R

```diff
@@ -0,0 +1,80 @@
+#' Compute interaction indicators for tspa() function
```

Comment:
It should be clarified that double-mean-centering is used. Documentation should include more technical details!

Reply:
A sentence for DMC is added